### ISSUE TITLE
Community page (users)

### ISSUE THREAD

**Issue by cwant** [ualbertalib/jupiter#229]
The wireframes are here:

![Browse Mockup](https://jupiter.mybalsamiq.com/mockups/6035142.png)
https://jupiter.mybalsamiq.com/projects/jupiter/Communities%20and%20Collections

A lot could be taken from (or shared with the) admin community page:

https://jupiter.mybalsamiq.com/projects/jupiter-admin/Communities%20and%20Collections

DONE WHEN:

- [x] The list of communities paginates
- [x] Sorting collections A-Z and Z-A works
- [x] A button toggles the display of a collection list for each community shown #246 

Near future work:
- [x] Auto-complete partial matches communities/collections as shown #242

**Comment by cwant** [comment]
The third item in the list is blocked by PR #246 (after that is merged, the collection dropdown can be copied from admin communities).
The last item (autocomplete) can be worked on separately (covered in Issue #242)

**Comment by cwant** [comment]
Last checkmark filled in, closing.
